app/components/admin/edition_images/uploaded_images_component.rb

```diff
@@ -3,36 +3,35 @@
 class Admin::EditionImages::UploadedImagesComponent < ViewComponent::Base
   def initialize(edition:)
     @edition = edition
-    @lead_image = has_lead_image? ? @edition.images.first : nil
-    @document_images = @edition.images.drop(has_lead_image? ? 1 : 0)
   end
 
-  def has_lead_image?
-    can_have_lead_image? && @edition.images.any?
+  def has_lead_image
```

Comment:
yea i'd prefer it but rubocop won't let you have a memoised variable different to the method name. Since it's used  a few times i figured it was probs worth the trade off. Happy to change it though if peiple disagree

Reply:
Oh that's interesting, fair enough